doc/rst/source/cookbook/features.rst

```diff
@@ -440,7 +440,13 @@ do not need to repeat the region and projection information, as shown here::
      gmt end show
 
 Thus, the chosen options remain in effect until you provide new option
-arguments on the command line.
+arguments on the command line.  **Note**: We keep track of two types of regions,
+One is the domain used for a map and one is the domain used for a grid.
+which often are the same.  When a plot is specified without providing
+a region then we look for a previous plot region in the history first, and
+if it is not found then we look for a grid domain to use instead.  However,
```

Comment:
So based on the grid domain description, if a plot region is not found a table region will not work (e.g., blockmean)?

Reply:
What I mean to say is that RG will be used if there is no RP, but RP will never be used if there is no RG.